.pre-commit-config.yaml

```diff
@@ -36,4 +36,4 @@ repos:
             language: node
             entry: ./frontend/node_modules/eslint/bin/eslint.js --max-warnings 0
             require_serial: true
-            files: \.(js|ts|svelte)$
+            files: frontend/.*\.(js|ts|svelte)$
```

Comment:
I was having trouble getting the linter to run properly on the extension code. Initially I was seeing:
`ESLint looked for configuration files in fava/src/fava/ext/portfolio_list and its ancestors. If it found none, it then looked in your home directory.`
I tried moving the .eslintrc.js file to the root of the project from frontend, but then I got:
`ESLint couldn't find the config "airbnb-base" to extend from. Please check that the name of the config is correct.
The config "airbnb-base" was referenced from the config file in "/home/tschicke/dev/projects/fava/.eslintrc.js".`
Moving node_modules as well doesn't fix the issue, and causes a few other issues related to tsconfig.json, and I didn't think making another small node project with another node_modules to handle the extension code was worth it for some example code.
I agree it would be nice to lint the extension code, so if you have a simpler way to get that working I can try to do that.

Reply:
Thanks for the attempt, this doesn't seem like it's worth the effort.